src/runtime/index.ts

```diff
@@ -1,5 +1,6 @@
 import {GLOBAL_SYMBOL} from '../common';
 import {TabsController} from './TabsController';
+import './scss/tabs.scss';
```

Comment:
Services will have to add `unsafe-inline` in the CSP config. And it breaks the backward compatibility too.

Reply:
I think it is bad to ask users to add unsafe CSP rules. We should somehow proxy nonce in the current case.

Reply:
There in no problem with csp. This is not inline css.